.github/workflows/linux.yml

```diff
@@ -28,10 +28,19 @@ jobs:
             std: 17
             install: sudo apt install g++-8
             os: ubuntu-18.04
+          - cxx: g++-9
+            build_type: Debug
+            std: 20
+            install: sudo apt install g++-9
+            os: ubuntu-18.04
           - cxx: g++-10
             build_type: Debug
             std: 17
             os: ubuntu-18.04
+          - cxx: g++-10
+            build_type: Debug
+            std: 20
+            os: ubuntu-18.04
```

Comment:
As commented earlier, let's not add new CI configs.

Reply:
Sorry, yes -- I'm just crap at git.  Will try again.